engine/deployments/docker-compose/1m1e.yaml

```diff
@@ -8,27 +8,19 @@ services:
                 command:
                         - "/tiflow"
                         - "master"
-                        - "--config"
-                        - "/config.toml"
+                        - "--config=/config.toml"
                         - "--log-level=debug"
-                        - "--master-addr"
-                        - "0.0.0.0:10240"
-                        - "--advertise-addr"
-                        - "server-master:10240"
-                        - "--peer-urls"
-                        - "http://127.0.0.1:8291"
-                        - "--advertise-peer-urls"
-                        - "http://server-master:8291"
-                        - "--frame-meta-endpoints"
-                        - "frame-mysql-standalone:3306"
-                        - "--business-meta-endpoints"
-                        - "business-etcd-standalone:2379"
+                        - "--master-addr=0.0.0.0:10240"
```

Comment:
Ref https://github.com/pingcap/tiflow/pull/6455. The command line parameters should be changed.

Reply:
Done.